src/game/RandomizationOfShipsPositions.js

```diff
@@ -0,0 +1,152 @@
+import { SHIPS_LIST } from "./constants";
+
+const GAME_BOARD = document.querySelectorAll(".board__fields");
+
+let gameBoardArray = [];
+let fieldsNamesArr = [];
+let direction;
+let position;
+const GAME_BOARD_WIDTH = 10;
+
+const createFieldsName = () => {
+  for (let i = 0; i < 10; i++) {
+    for (let j = 0; j < 10; j++) {
+      fieldsNamesArr.push(`${String.fromCharCode(i + 65)}${j + 1}`);
+    }
+  }
+};
+createFieldsName();
+
+const createGameBoardArray = () => {
+  gameBoardArray = [];
+  for (let i = 0; i < GAME_BOARD_WIDTH; i++) {
+    gameBoardArray.push([]);
+    for (let j = 0; j < GAME_BOARD_WIDTH; j++) {
+      gameBoardArray[i].push({
+        isEmpty: true,
+        shipName: "",
+      });
+    }
+  }
+};
+
+const directionRoll = () => {
+  Math.floor(Math.random() * 2) === 0
```

Comment:
I would rename it `rollDirection` and I suggest a code like that (after removing `let direction` earlier):
`
const rollDirection= () => {
  return Math.floor(Math.random() * 2) === 0 ? "vertical" : "horizontal";
}
`